android/src/main/kotlin/com/qonversion/flutter/sdk/qonversion_flutter_sdk/QonversionPlugin.kt

```diff
@@ -263,6 +272,21 @@ class QonversionPlugin : MethodCallHandler, FlutterPlugin, ActivityAware {
         qonversionSandwich.checkTrialIntroEligibility(ids, result.toJsonResultListener())
     }
 
+    private fun attachUserToExperiment(args: Map<String, Any>, result: Result) {
+        @Suppress("UNCHECKED_CAST")
```

Comment:
Does it really warn here (and below)? Seems like it shoudn't

Reply:
It's a copy/paste from a similar function. I'll check and remove (if needed) it later.